test/7.js

```diff
@@ -1,7 +1,8 @@
 const expect = require('chai').expect;
-// require your getLongestInc function
+const arrayOp = require('../solutions/7');
+const getLongestInc = arrayOp.getLongestInc;
 
-/* describe('getLongestInc', () => {
+describe('getLongestInc', () => {
     it('should return the number of longest increasing numbers in an array', () => {
       let arr = [2,3];
```

Comment:
test cases needs to be grouped and separated. for example:
it should return the correct result when middle number is decreasing